pkg/config/agent/config.go

```diff
@@ -208,6 +208,8 @@ type AgentConfig struct {
 	// second(pps) and the burst size will be automatically set to twice the rate.
 	// When the rate and burst size are exceeded, new packets will be dropped.
 	PacketInRate int `yaml:"packetInRate,omitempty"`
+	// HostNetworkAcceleration configures acceleration of traffic in Node host network.
```

Comment:
Does nfttables require newer kernel version? Is it by default enabled in kernel?

Reply:
I checked. Antrea already requires >4.6 kernel version or CentOS 7.4 which supports nftables too. Then no need to add the kernel version requirement for this option.